packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -280,6 +280,7 @@ registerFragment(`
     contents {
       _id
       htmlHighlight
+      plaintextDescription
```

Comment:
This may have a slight negative performance impact. I don't think it'll be too bad, but it's worth keeping an eye out.